seed/tests/test_meter_views.py

```diff
@@ -2,17 +2,23 @@
 # encoding: utf-8
 
 import ast
+import copy
+import json
 
 from django.urls import reverse
 
-from seed.data_importer.utils import kbtu_thermal_conversion_factors
+from seed.data_importer.utils import \
+    kbtu_thermal_conversion_factors as conversion_factors
 from seed.landing.models import SEEDUser as User
-from seed.tests.util import DataMappingBaseTestCase
+from seed.test_helpers.fake import FakePropertyViewFactory
+from seed.tests.util import DeleteModelsTestCase
 from seed.utils.organizations import create_organization
 
 
-class TestMeterValidTypesUnits(DataMappingBaseTestCase):
+class TestMeterValidTypesUnits(DeleteModelsTestCase):
```

Comment:
Why this change?

Reply:
`DataMappingBaseTestCase` has a bunch of helper methods for running the mapping and merging. It inherits from `DeleteModelsTestCase` which simply makes sure that all the models are empty when the test suite runs.